### ISSUE TITLE
Sonatype reporting critical vulnerability in 1.6.49

### ISSUE THREAD

**Issue by ragamuffin-coder** [peterolson/BigInteger.js#220]
Latest release v1.6.49 is being flagged as having a [critical vulnerability by Sonatype](https://ossindex.sonatype.org/component/pkg:npm/big-integer@1.6.49?utm_source=dependency-check&utm_medium=integration&utm_content=6.3.1).

The exact vulnerability reported through [OWASP dependency](https://owasp.org/www-project-dependency-check/) check is as follows:
https://ossindex.sonatype.org/vulnerability/afbfcdda-fd2d-42b6-aa10-bf8343466d99

**Comment by gardhr** [comment]
Can you be a little more specific? Because the links you've provided don't seem to mention the exact dependency from which the vulnerability supposedly originates.

**Comment by attritionorg** [comment]
Yeah, had to follow the rabbit hole. Sonatype should directly reference:
http://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-23337

**Comment by peterolson** [comment]
It says "Lodash versions prior to 4.17.21 are vulnerable to Command Injection via the template function." but the dependency used is version 4.17.21 (and it's only a devDependency). I'm not sure what the issue is?
